fastlane/lib/fastlane/actions/docs/create_app_online.md

```diff
@@ -101,6 +104,24 @@ If you want to associate an app with an App Group:
 fastlane produce associate_group -a com.krausefx.app group.krausefx
 ```
 
+If you want to create a new iCloud Container:
+
+```no-highlight
+fastlane produce cloud_container -ci iCloud.com.krausefx.app -n "Example iCloud Container"
+```
+
+If you want to associate an app with an iCloud Container:
+
+```no-highlight
+fastlane produce associate_cloud_container -a com.krausefx.app iCloud.com.krausefx.app
+```
+
+If you want to associate an app with multiple iCloud Containers:
+
+```no-highlight
+fastlane produce associate_merchant -a com.krausefx.app iCloud.com.krausefx.app1 iCloud.com.krausefx.app2
```

Comment:
Is this `associate_merchant` a copy/paste error? I think this should be `associate_cloud_container`?

Reply:
@joshdholtz You're right. Never good to be lazy 😉 It's fixed now!